<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="selectFileButton">
      <property name="text">
       <string>Select File</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="keyLineEdit">
      <property name="placeholderText">
       <string>Enter Key</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="setKeyButton">
      <property name="text">
       <string>Select Key</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="valueLineEdit">
      <property name="placeholderText">
       <string>Enter Value</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="setValueButton">
      <property name="text">
       <string>Set Value</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>selectFileButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>selectFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>setValueButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>setValue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectFile()</slot>
  <slot>setValue()</slot>
 </slots>
</ui>
